src/test/java/com/github/ozsie/CheckMojoSpec.kt

```diff
@@ -1,22 +1,14 @@
 package com.github.ozsie
 
+import com.github.ozsie.test.CheckMojoTestFactory
 import io.github.detekt.tooling.api.MaxIssuesReached
 import org.jetbrains.spek.api.Spek
 import org.jetbrains.spek.api.dsl.given
 import org.jetbrains.spek.api.dsl.on
-import java.net.URI
-import java.nio.file.Paths
 import kotlin.test.assertFailsWith
 import kotlin.test.expect
 
-
 class CheckMojoSpec : Spek({
-    val invalidPackageNamingDirectoryPath by lazy {
```

Comment:
Extracted a test factors to separate concerns, reduce duplication and improve readability